app/src/main/kotlin/com/wire/android/ui/userprofile/UserProfileViewModel.kt

```diff
@@ -165,8 +178,7 @@ class UserProfileViewModel @Inject constructor(
         }
     }
 
-    // !! TODO: this method is made only to pass the mock bitmap, later on we will not need it !!
-    fun mockMethodForAvatar(bitmap: Bitmap) {
-        userProfileState = userProfileState.copy(avatarBitmap = bitmap)
+    companion object {
+        const val GENERIC_DOWNLOAD_USER_INFO_ERROR = 10
```

Comment:
how about passing Int value that do not have much meaning, we pass a sealed class or an enum ?